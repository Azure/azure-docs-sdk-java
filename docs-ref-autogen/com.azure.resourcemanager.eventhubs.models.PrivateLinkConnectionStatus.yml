### YamlMime:JavaType
uid: "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus"
fullName: "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus"
name: "PrivateLinkConnectionStatus"
nameWithType: "PrivateLinkConnectionStatus"
summary: "Defines values for PrivateLinkConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateLinkConnectionStatus extends ExpandableStringEnum<PrivateLinkConnectionStatus>"
constructors:
- "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.PrivateLinkConnectionStatus()"
fields:
- "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.APPROVED"
- "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.DISCONNECTED"
- "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.PENDING"
- "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.REJECTED"
methods:
- "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.eventhubs.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-eventhubs:2.13.0
