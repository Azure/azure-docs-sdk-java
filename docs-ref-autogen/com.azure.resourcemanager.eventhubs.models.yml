### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.eventhubs.models"
fullName: "com.azure.resourcemanager.eventhubs.models"
name: "com.azure.resourcemanager.eventhubs.models"
summary: "Package containing the data models for EventHubManagementClient."
classes:
- "com.azure.resourcemanager.eventhubs.models.AccessRights"
- "com.azure.resourcemanager.eventhubs.models.ArmDisasterRecoveryListResult"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRuleListResult"
- "com.azure.resourcemanager.eventhubs.models.AvailableCluster"
- "com.azure.resourcemanager.eventhubs.models.CaptureDescription"
- "com.azure.resourcemanager.eventhubs.models.CheckNameAvailabilityParameter"
- "com.azure.resourcemanager.eventhubs.models.ClusterListResult"
- "com.azure.resourcemanager.eventhubs.models.ClusterSku"
- "com.azure.resourcemanager.eventhubs.models.ClusterSkuName"
- "com.azure.resourcemanager.eventhubs.models.ConnectionState"
- "com.azure.resourcemanager.eventhubs.models.ConsumerGroupListResult"
- "com.azure.resourcemanager.eventhubs.models.DefaultAction"
- "com.azure.resourcemanager.eventhubs.models.Destination"
- "com.azure.resourcemanager.eventhubs.models.EHNamespaceIdContainer"
- "com.azure.resourcemanager.eventhubs.models.EHNamespaceListResult"
- "com.azure.resourcemanager.eventhubs.models.Encryption"
- "com.azure.resourcemanager.eventhubs.models.EndPointProvisioningState"
- "com.azure.resourcemanager.eventhubs.models.EventHubListResult"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaceSkuType"
- "com.azure.resourcemanager.eventhubs.models.Identity"
- "com.azure.resourcemanager.eventhubs.models.KeyType"
- "com.azure.resourcemanager.eventhubs.models.KeyVaultProperties"
- "com.azure.resourcemanager.eventhubs.models.NWRuleSetIpRules"
- "com.azure.resourcemanager.eventhubs.models.NWRuleSetVirtualNetworkRules"
- "com.azure.resourcemanager.eventhubs.models.NetworkRuleIpAction"
- "com.azure.resourcemanager.eventhubs.models.OperationDisplay"
- "com.azure.resourcemanager.eventhubs.models.OperationListResult"
- "com.azure.resourcemanager.eventhubs.models.PrivateEndpoint"
- "com.azure.resourcemanager.eventhubs.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus"
- "com.azure.resourcemanager.eventhubs.models.PrivateLinkResource"
- "com.azure.resourcemanager.eventhubs.models.PublicNetworkAccessFlag"
- "com.azure.resourcemanager.eventhubs.models.RegenerateAccessKeyParameters"
- "com.azure.resourcemanager.eventhubs.models.SchemaCompatibility"
- "com.azure.resourcemanager.eventhubs.models.SchemaGroupListResult"
- "com.azure.resourcemanager.eventhubs.models.SchemaType"
- "com.azure.resourcemanager.eventhubs.models.Sku"
- "com.azure.resourcemanager.eventhubs.models.SkuName"
- "com.azure.resourcemanager.eventhubs.models.SkuTier"
- "com.azure.resourcemanager.eventhubs.models.Subnet"
- "com.azure.resourcemanager.eventhubs.models.UserAssignedIdentity"
- "com.azure.resourcemanager.eventhubs.models.UserAssignedIdentityProperties"
enums:
- "com.azure.resourcemanager.eventhubs.models.EncodingCaptureDescription"
- "com.azure.resourcemanager.eventhubs.models.EntityStatus"
- "com.azure.resourcemanager.eventhubs.models.KeySource"
- "com.azure.resourcemanager.eventhubs.models.ManagedServiceIdentityType"
- "com.azure.resourcemanager.eventhubs.models.ProvisioningStateDR"
- "com.azure.resourcemanager.eventhubs.models.RoleDisasterRecovery"
- "com.azure.resourcemanager.eventhubs.models.UnavailableReason"
interfaces:
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.DefinitionStages"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.DefinitionStages.WithListen"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.DefinitionStages.WithListenOrSendOrManage"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.DefinitionStages.WithManage"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.DefinitionStages.WithSend"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.DefinitionStages.WithSendOrManage"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.UpdateStages"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.UpdateStages.WithListen"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.UpdateStages.WithListenOrSendOrManage"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.UpdateStages.WithManage"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.UpdateStages.WithSend"
- "com.azure.resourcemanager.eventhubs.models.AuthorizationRule.UpdateStages.WithSendOrManage"
- "com.azure.resourcemanager.eventhubs.models.DisasterRecoveryPairingAuthorizationKey"
- "com.azure.resourcemanager.eventhubs.models.DisasterRecoveryPairingAuthorizationRule"
- "com.azure.resourcemanager.eventhubs.models.DisasterRecoveryPairingAuthorizationRules"
- "com.azure.resourcemanager.eventhubs.models.EventHub"
- "com.azure.resourcemanager.eventhubs.models.EventHub.Definition"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages.Blank"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages.WithAuthorizationRule"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages.WithCaptureEnabledDisabled"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages.WithCaptureOptionalSettingsOrCreate"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages.WithCaptureProviderOrCreate"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages.WithConsumerGroup"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages.WithNamespace"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages.WithPartitionCount"
- "com.azure.resourcemanager.eventhubs.models.EventHub.DefinitionStages.WithRetentionPeriod"
- "com.azure.resourcemanager.eventhubs.models.EventHub.Update"
- "com.azure.resourcemanager.eventhubs.models.EventHub.UpdateStages"
- "com.azure.resourcemanager.eventhubs.models.EventHub.UpdateStages.WithAuthorizationRule"
- "com.azure.resourcemanager.eventhubs.models.EventHub.UpdateStages.WithCapture"
- "com.azure.resourcemanager.eventhubs.models.EventHub.UpdateStages.WithConsumerGroup"
- "com.azure.resourcemanager.eventhubs.models.EventHub.UpdateStages.WithPartitionCount"
- "com.azure.resourcemanager.eventhubs.models.EventHub.UpdateStages.WithRetentionPeriod"
- "com.azure.resourcemanager.eventhubs.models.EventHubAuthorizationKey"
- "com.azure.resourcemanager.eventhubs.models.EventHubAuthorizationRule"
- "com.azure.resourcemanager.eventhubs.models.EventHubAuthorizationRule.Definition"
- "com.azure.resourcemanager.eventhubs.models.EventHubAuthorizationRule.DefinitionStages"
- "com.azure.resourcemanager.eventhubs.models.EventHubAuthorizationRule.DefinitionStages.Blank"
- "com.azure.resourcemanager.eventhubs.models.EventHubAuthorizationRule.DefinitionStages.WithAccessPolicy"
- "com.azure.resourcemanager.eventhubs.models.EventHubAuthorizationRule.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.eventhubs.models.EventHubAuthorizationRule.DefinitionStages.WithEventHub"
- "com.azure.resourcemanager.eventhubs.models.EventHubAuthorizationRule.Update"
- "com.azure.resourcemanager.eventhubs.models.EventHubAuthorizationRules"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroup"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroup.Definition"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroup.DefinitionStages"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroup.DefinitionStages.Blank"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroup.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroup.DefinitionStages.WithEventHub"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroup.DefinitionStages.WithUserMetadata"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroup.Update"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroup.UpdateStages"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroup.UpdateStages.WithUserMetadata"
- "com.azure.resourcemanager.eventhubs.models.EventHubConsumerGroups"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairing"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairing.Definition"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairing.DefinitionStages"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairing.DefinitionStages.Blank"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairing.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairing.DefinitionStages.WithPrimaryNamespace"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairing.DefinitionStages.WithSecondaryNamespace"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairing.Update"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairing.UpdateStages"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairing.UpdateStages.WithSecondaryNamespace"
- "com.azure.resourcemanager.eventhubs.models.EventHubDisasterRecoveryPairings"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.Definition"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.DefinitionStages"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.DefinitionStages.Blank"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.DefinitionStages.WithAuthorizationRule"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.DefinitionStages.WithEventHub"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.DefinitionStages.WithGroup"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.DefinitionStages.WithSku"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.DefinitionStages.WithThroughputConfiguration"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.Update"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.UpdateStages"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.UpdateStages.WithAuthorizationRule"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.UpdateStages.WithEventHub"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.UpdateStages.WithSku"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespace.UpdateStages.WithThroughputConfiguration"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaceAuthorizationRule"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaceAuthorizationRule.Definition"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaceAuthorizationRule.DefinitionStages"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaceAuthorizationRule.DefinitionStages.Blank"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaceAuthorizationRule.DefinitionStages.WithAccessPolicy"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaceAuthorizationRule.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaceAuthorizationRule.DefinitionStages.WithNamespace"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaceAuthorizationRule.Update"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaceAuthorizationRules"
- "com.azure.resourcemanager.eventhubs.models.EventHubNamespaces"
- "com.azure.resourcemanager.eventhubs.models.EventHubs"
- "com.azure.resourcemanager.eventhubs.models.NestedResource"
desc: "Package containing the data models for EventHubManagementClient. Azure Event Hubs client for managing Event Hubs\n Cluster, IPFilter Rules and VirtualNetworkRules resources."
metadata: {}
package: "com.azure.resourcemanager.eventhubs.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-eventhubs:2.19.0
