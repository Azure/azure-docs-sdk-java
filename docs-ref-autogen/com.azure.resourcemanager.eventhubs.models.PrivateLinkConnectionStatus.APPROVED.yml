### YamlMime:JavaMember
uid: "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.eventhubs.models.PrivateLinkConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkConnectionStatus."
  syntax: "public static final PrivateLinkConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.eventhubs.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-eventhubs:2.13.0
